compiler/semantics.cpp

```diff
@@ -2307,6 +2307,18 @@ bool Semantics::CheckExprStmt(ExprStmt* stmt) {
     return true;
 }
 
+bool Semantics::IsIncluded(Decl* sym) {
+    const auto fileno = cc().sources()->GetSourceFileIndex(sym->pos());
+    if (fileno >= cc().sources()->opened_files().size()) { return false; }
```

Comment:
nit: no single-line conditionals. should be:

    if (fileno >= cc().sources()->opened_files().size()
        return false;

Though, I think this condition is unnecessary. We should never get a source index for a file not in the open list.